src/app/api/env.coffee

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  API_URL: 'http://localhost:3000/api'
```

Comment:
I forgot to add the copyenv to scripts where needed and remove this file from git. 
the copyenv script will detect NODE_ENV and add relevant config, so:

```
npm run dev
```
will copy over config/environment/development.env.coffee

```
env NODE_ENV=production npm run dev
```
will copy over config/environment/production.env.coffee

you can set the NODE_ENV before any npm script